### PR TITLE
move defaultskill to a config menu item

### PR DESCRIPTION
**Pull request by fabiangreffrath**

### CODE REVIEW

File: Source/m_menu.c [hunk 1]
```diff
@@ -3725,6 +3723,9 @@ setup_menu_t gen_settings2[] = { // General Settings screen2
   {"Game speed, percentage of normal", S_NUM|S_PRGWARN, m_null, G_X,
    G_Y4 + general_realtic*8, {"realtic_clock_rate"}},
 
+  {"Default skill level", S_CHOICE|S_LEVWARN, m_null, G_X,
+   G_Y4 + general_skill*8, {"default_skill"}, 0, NULL, default_skill_strings},
+
```
[the remark this answers is not in the document]
Comment (rfomin) [reply Source/m_menu.c:3728]: I like removing "Files Preloaded at Game Startup", can we do it anyway?

**Review by rfomin — APPROVED** [review]

File: Source/m_menu.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (rfomin) [reply Source/m_menu.c:3728]: We coud remove/move to second page "Flashing HOM indicator" and "Flash Icon During Disk IO"

[the remark this answers is not in the document]
Comment (fabiangreffrath) [reply Source/m_menu.c:3728]: Number of player corpses 🤷

[the remark this answers is not in the document]
Comment (rfomin) [reply Source/m_menu.c:3728]: This way we can cut everything down to one page, or move miscellaneous stuff to a second page.

**Pull request merged**